src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+         [34.03575887721561, -12.127781090999603],
+         [34.10030355494999, -12.137179413699451],
+         [34.024805840446966, -12.23935311768831],
+         [34.01948433776142, -12.243547074452556],
+         [34.01716690917255, -12.234655807051144],
+         [33.997082528069036, -12.237088353343825],
+         [33.99871331115009, -12.247740964599842],
+         [34.00180321593525, -12.242372773272642],
+         [33.99456186309381, -12.239664724585708],
+         [33.985618444535525, -12.233731858979626],
+         [33.96905312165955, -12.24178436852256],
+         [33.95703682527283, -12.237925904996903],
+         [33.95257362947205, -12.241700489480108],
+         [33.95102867707947, -12.228866682206075],
+         [33.93823990449646, -12.225259683004753],
+         [33.93206009492615, -12.23566112841648],
+         [33.93497833833435, -12.24983663252191],
+         [33.9498270474408, -12.248997866483007],
+         [33.9142931424115, -12.228363382946103],
+         [33.911718221757205, -12.242623157481672],
+         [33.98742088899353, -12.254869172764044],
+         [33.953431936356814, -12.262921037431857],
+         [33.95806679353455, -12.220142692759165]]),
+    Sentinel2 = ee.Image("projects/ee-aasareansah/assets/Sent_Sept_Comp_Malawi"),
+    Malawi_EPA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_EPAs"),
+    Malawi_TA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_TAs"),
+    Admin = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_admin2");
+
+
+//Rename b1 as True value 
+var upd_name = malawi_binary.select('b1').rename('Wrong value');
+
+var leftMap = ui.Map();
+var layer1 = ui.Map.GeometryLayer([], 'Geometry Layer', 'red');
+var drawingTools1 = leftMap.drawingTools();
+drawingTools1.layers().add(layer1);
+
+var rightMap = ui.Map();
+var layer2 = ui.Map.Layer(ee.FeatureCollection([]), {'color': 'red'}, 'Points');
+
+// Link two layers together
+drawingTools1.onDraw(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+drawingTools1.onEdit(ui.util.debounce(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+}, 100));
+drawingTools1.onErase(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+
+//Set map center using this long lat and zoom level
+rightMap.setCenter(33.78, -12.27, 11);
+
+// Make the default basemap the satellite layer
+rightMap.setOptions('satellite');
+
+//Specify the point as the only drawing tool 
+drawingTools1.setDrawModes(['point']);
+
+//Assign specific palettes to the cropland map
+var palettes = require('users/gena/packages:palettes');
+var palette = palettes.cmocean.Speed[7];
+
+// Use this empty image for paint().
+var empty = ee.Image().byte();
+
+var Admin_color = empty.paint({
+featureCollection: Admin,
+color: 1,
+width: 2
+});
```

Comment:
I think these colors should be written as one line to condense the code a bit. Also generally javascript uses camelCase for variable names so I would suggest being consistent with the convention.
```js
var adminColor = empty.paint({featureCollection: Admin, color: 1, width: 2});
```